### ISSUE TITLE
String viewer to view long string

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#1259]

**Comment by akhil117** [comment]
It means we can enter the array of strings and with the help of string viewer we should know the long string name ?
Is it?

**Comment by mrexodia** [comment]
No, it would be a simple dialog that is opened when you right click in the dump that allows you to view and copy the long string.

For example:

![](https://i.imgur.com/VXzSHT1.png)

Right click on that byte, 'view string' and then it will show a multi-line text editor that allows you to interact with the string. Perhaps the Data Copy dialog could be used to display the string:

![](https://i.imgur.com/zlwLwq4.png)
